app/screens/main/Main.js

```diff
@@ -139,82 +139,82 @@ class Main extends Component<Props, State> {
         <Logo />
         <InnerWrapper>
           <NavBar>
-          <NavBarPart>
-            <NavLinksWrapper>
+            <NavBarPart>
```

Comment:
we've talked multiple times about submitting PR without introducing distraction. Why do you submit those changes?

Reply:
Auto formatter does this... I didn't really change anything here.

Reply:
So you are not reviewing your PRs after submission, something we talked about a lot